browser/src/control/Control.Menubar.js

```diff
@@ -2019,6 +2019,13 @@ L.Control.Menubar = L.Control.extend({
 	_createMenu: function(menu) {
 		var itemList = [];
 		var docType = this._map.getDocType();
+		var isReadOnly = this._map.isPermissionReadOnly();
+		var isPermissionEditForComments = this._map.isPermissionEditForComments(); // Checks if PDF or not indirectly.
```

Comment:
current implementation of `isPermissionEditForComments`
is used to detect that we can add comments on mobile when not in "Edit mode" - so it means that actually file is writable:
https://github.com/CollaboraOnline/online/blob/7e22a01d4d0a2fe026a4b89f31678706d6148b6f/browser/src/control/Permission.js#L248